mysql/datadog_checks/mysql/activity.py

```diff
@@ -135,6 +135,18 @@ def __init__(self, check, config, connection_args):
 
     def run_job(self):
         # type: () -> None
+        # Detect a database misconfiguration by checking if `events-waits-current` is enabled.
+        if not self._check.events_wait_current_enabled:
+            self._check.record_warning(
+                DatabaseConfigurationError.events_waits_current_not_enabled,
+                warning_with_tags(
+                    'Unable to collect query activity because `performance-schema-consumer-events-waits-current` '
+                    'is not enabled.',
```

Comment:
```suggestion
                    'Query activity and wait event collection is disabled on this host. To enable it, the setup consumer `performance-schema-consumer-events-waits-current` must be enabled on the MySQL server. For help with enabling this consumer, please see the troubleshooting guide: <insert here>',
```

Reply:
Nice, although question regarding this piece.
> For help with enabling this consumer, please see the troubleshooting guide: <insert here>
What are your thoughts on adding this on the front-end rather than hard coding some troubleshooting guide URL in the integration? I was hesitant on adding it here since this could eventually change. It's not as convenient when looking at the logs, but I think providing the exact setup consumer in the message is good enough?

Reply:
Ok in this case since we are adding in-app warnings and we don't have the permalink ready yet, I think it is okay. But the text should read:

`Please refer to the troubleshooting documentation: https://docs.datadoghq.com/database_monitoring/setup_mysql/troubleshooting/`

If there is no action for the customer to take, it will be very confusing to read this message.